apps/vaporgui/MainForm.cpp

```diff
@@ -395,18 +396,30 @@ MainForm::MainForm(
 	}
 
     
-	if (files.size() && files[0].endsWith(".nc")) {
-        VDCNetCDF vdc;
-        bool errReportingEnabled = Wasp::MyBase::EnableErrMsg(false);
-        int ret = vdc.Initialize(files[0].toStdString(), {}, VDC::R);
-        Wasp::MyBase::EnableErrMsg(errReportingEnabled);
-        if (ret < 0) {
-            loadDataHelper({files[0].toStdString()}, "NetCDF CF files", "", "cf", true);
-        } else {
+	if (files.size()) {
+        if (files[0].endsWith(".nc")) {
```

Comment:
@stasj it's great to have this, but we may want to think about changing the file type detection logic a bit. Consider:

1. NetCDF CF files usually have .nc extensions, but not always
2. WRF files usually do not have any extension, but sometimes .nc is used
3. The DC*::Initialize() method should work reliably for detecting wrf files, VDC files, AND MPAS files, but DCCF::Initialize() will succeed for VDC, WRF, and MPAS files. 

Given that I would suggest NOT using extensions and testing for file types using DC*::Initialize() in this order:

VDC, WRF, MPAS, CF

As far as order goes the only one that is really important is to have CF NetCDF last. 

What do you think?

Reply:
I see, I can make that change. I included the file extensions here to mildly enforce the naming conventions however you seem to be saying that they are all over the place. Therefore I think it makes sense to ignore the file extension entirely and determine the file type based on DC*::Initialize().

Reply:
Done. We could consider replacing `Open VDC` and `Import` with just `Open Dataset` and use the method used here.